class LeadAlreadyExistsError(Exception):
    pass


class LeadNotFoundError(Exception):
    pass
